app/Settings/TicketSettings.php

```diff
@@ -6,7 +6,7 @@
 
 class TicketSettings extends Settings
 {
-    public bool $enabled;
+    public ?bool $enabled;
```

Comment:
Why is this optional? A boolean should not be nullable but rather have a default value

Reply:
I dont know this is not a change from me, its a change from jameskitt if I remember well on te original installer fix branch

Reply:
yep commit [Fix wrong php typehint](https://github.com/Ctrlpanel-gg/panel/pull/972/commits/9772321a4d426f729cc90b2779ff8b783793339d)

@jameskitt616

Reply:
Needs to be resolved, a boolean cannot be nullable.

Reply:
fixed